libs/src/api/user.js

```diff
@@ -501,27 +416,33 @@ class Users extends Base {
     // Populate metadata with required fields - wallet, is_creator, creator_node_endpoint
     newMetadata.wallet = this.web3Manager.getWalletAddress()
     newMetadata.is_creator = true
-    newMetadata.creator_node_endpoint = newCreatorNodeEndpoint
-
-    const newPrimary = CreatorNode.getPrimary(newCreatorNodeEndpoint)
-
-    // Sync user data from old primary to new endpoint
-    if (existingEndpoint) {
-      // Don't validate what we're syncing from because the user isn't
-      // a creator yet.
-      await this.creatorNode.syncSecondary(
-        newPrimary,
-        existingEndpoint,
-        /* immediate= */ true,
-        /* validate= */ false
-      )
-    }
 
-    // Update local libs state with new CN endpoint
-    await this.creatorNode.setEndpoint(newPrimary)
+    if (oldMetadata.creator_node_endpoint) {
+      // Update the newMetadata with the existing creator_node_endpoint field from oldMetadata
+      newMetadata.creator_node_endpoint = oldMetadata.creator_node_endpoint
```

Comment:
is this neceessary? given that `newMetadata` is defined as `const newMetadata = this._cleanUserMetadata({ ...user })` on line 413, isn't it guaranteed to always have the endpoint

Reply:
find out why USER_PROPS has the fields it does / where is wallet and user_id

Reply:
for this convo, don't need if block